### PR TITLE
I18n devel translations

### PR DESCRIPTION
**Pull request by rooftopcellist**
##### SUMMARY

First round of translations for ui_next.  I have also included:
* necessary changes to import fr, nl, es, zh, and also add them to the catalog.  
* fixed a syntax error that was keeping strings from compiling.  

##### ISSUE TYPE

 - Feature

##### COMPONENT NAME

 - Translations

##### AWX VERSION

```
devel
```

### CODE REVIEW

File: awx/ui_next/src/App.jsx [hunk 1]
```diff
@@ -29,7 +33,7 @@ const ProtectedRoute = ({ children, ...rest }) =>
   );
 
 function App() {
-  const catalogs = { en, ja };
+  const catalogs = { en, es, fr, ja, nl, zh };
```
Comment (jakemcdermott) [inline awx/ui_next/src/App.jsx]: :+1: yep

**Review by jakemcdermott — APPROVED** [review]
lgtm - if all layers of tests are happy, I am happy

**Comment by rooftopcellist** [comment]
recheck

**Review by AlexSCorey — DISMISSED** [review]
There have been some significant changes in the way we set up ling ui in App.jsx and our syntax for marking strings for translation. Please rebase this PR to get the new way we set up I18n.

The new syntax for making strings for translation is ``` t`string` ```

**Comment by rooftopcellist** [comment]
Thanks for the heads up @AlexSCorey , I've rebased this PR.  There were some more recent extracted strings (still untranslated) in devel.  This PR may remove some of them, that is OK because when we go to push new strings to the translation team, we will get the newest strings available at that time.

**Comment by rooftopcellist** [comment]
CI is passing now.  Thanks for the help @AlexSCorey !

File: awx/ui_next/src/App.jsx [hunk 2]
```diff
@@ -17,12 +17,6 @@ import Background from './components/Background';
 import NotFound from './screens/NotFound';
 import Login from './screens/Login';
 
-import en from './locales/en/messages';
-import es from './locales/es/messages';
-import fr from './locales/fr/messages';
-import ja from './locales/ja/messages';
-import nl from './locales/nl/messages';
-import zh from './locales/zh/messages';
```
Comment (jakemcdermott) [inline awx/ui_next/src/App.jsx]: :+1: 

nice :)

**Review by jakemcdermott — APPROVED** [review]

File: awx/ui_next/src/App.jsx [hunk 3]
```diff
@@ -84,7 +78,6 @@ const ProtectedRoute = ({ children, ...rest }) =>
   );
 
 function App() {
-  const catalogs = { en, es, fr, ja, nl, zh };
```
Comment (jakemcdermott) [inline awx/ui_next/src/App.jsx]: :+1: 
nice :)

**Review by jakemcdermott — APPROVED** [review]

**Pull request merged**